controllers/account/test/README.md

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
Dir named `test` will be treated as normal code and complied into binaries, renamed dir name to `testdata` or rename all files to `*_test.go`. 
Refer: https://pkg.go.dev/cmd/go/internal/test

Reply:
rename dir `testdata`may be better,done it